app/main.py

```diff
@@ -1,2 +1,115 @@
+from __future__ import annotations
+from typing import List, Union, Any
+import traceback
+
+from app.point import Point
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.load_factor = 2 / 3
+        self.length = 0
+        self.capacity = 8
+        self.hash_table: List = [None] * self.capacity
+
+    def __str__(self) -> str:
+        return str(self.hash_table)
+
+    def __len__(self) -> int:
+        return self.length
+
+    def __iter__(self) -> Dictionary:
+        self.current_element = 0
+        return self
+
+    def __next__(self) -> Union[int, float, str, bool, tuple, Point]:
+        while self.hash_table[self.current_element] is None:
+            self.current_element += 1
+            if self.current_element > self.capacity - 1:
+                raise StopIteration
+        result = self.hash_table[self.current_element][0]
+        self.current_element += 1
+        return result
+
+    def __setitem__(
+            self,
+            key: Union[int, float, str, bool, tuple, Point],
+            value: Any
+    ) -> None:
+        hashed_key = hash(key)
+        index = hashed_key % self.capacity
+        while self.hash_table[index] is not None:
+            current_key, current_value = self.hash_table[index]
+            if current_key == key:
+                self.hash_table[index] = [current_key, value]
```

Comment:
Store hash value too

Reply:
when i store hash, my code doesn't pass the tests. I get "ValueError: too many values to unpack (expected 2)" in every single test